src/NosCore.GameObject/Providers/MinilandProvider/IMinilandProvider.cs

```diff
@@ -20,6 +20,8 @@
 using System;
 using System.Collections.Generic;
 using ChickenAPI.Packets.Enumerations;
+using NosCore.GameObject.HttpClients.FriendHttpClient;
+using NosCore.GameObject.Networking.ClientSession;
```

Comment:
useless

Reply:
True, i modify it.
Forgot to run Resharper.